pyproject.toml

```diff
@@ -23,6 +22,7 @@ pytest-httpx = "^0.22.0"
 langchain = "^0.0.326"
 openai = "^0.28.1"
 freezegun = "^1.2.2"
+wrapt = "^1.15.0"
```

Comment:
this is only used by the openai automated patch function (`trace_openai`) so moving it to dev dependencies and doing a runtime check like we are with openai